pytest_playwright/pytest_playwright.py

```diff
@@ -101,16 +122,31 @@ def browser_type_launch_args(pytestconfig: Any) -> Dict:
     return launch_options
 
 
-@pytest.fixture(scope="session")
+def _build_artifact_test_folder(
+    pytestconfig: Any, request: pytest.FixtureRequest, folder_or_file_name: str
+) -> str:
+    output_dir = pytestconfig.getoption("--output")
+    return os.path.join(output_dir, slugify(request.node.nodeid), folder_or_file_name)
+
+
+@pytest.fixture()
```

Comment:
This was a "session" fixture before - what's the difference?

Reply:
Before: Context options were for the whole test execution
After: Context options are per test. Since we re-create a context for every test, this should be a better handling.
See here for reference: https://docs.pytest.org/en/6.2.x/fixture.html#fixture-scopes

Reply:
I am a bit worried whether this will affect some users. For example, their custom fixture depends on this one, and now it cannot because of wrong scope? I see changes in our own tests that kind of confirm this?

Reply:
Yes, but since this is 0.x we are allowed to do that. Before it was more a wrong implementation from my side.